### PR TITLE
Feat/message box component

### PR DESCRIPTION
**Pull request by mpisacic**
MessageBox component

### CODE REVIEW

File: src/components/MessageBox/MessageBox.tsx [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+                    <Icon iconName={iconName} />
+                    {message}
+                </div>
+                {
+                    isLoading && <Spinner className="message-box-spinner" />
+                }
+                {
+                    errorMessage !== undefined && errorMessage !== '' &&
+                    <Tooltip containerClass="operation-error-tooltip-container" content={errorMessage} directionalHint={DirectionalHint.rightCenter}>
+                        <Icon className="operation-error" iconName="icon-usklicnik" />
```
Comment (hbagarich) [inline src/components/MessageBox/MessageBox.tsx]: can this iconName be either a prop or something more appropriate?

File: src/components/MessageBox/MessageBox.tsx [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+                <div className="message-box-container">
+                    <Icon iconName={iconName} />
+                    {message}
+                </div>
+                {
+                    isLoading && <Spinner className="message-box-spinner" />
+                }
+                {
+                    errorMessage !== undefined && errorMessage !== '' &&
+                    <Tooltip containerClass="operation-error-tooltip-container" content={errorMessage} directionalHint={DirectionalHint.rightCenter}>
```
Comment (hbagarich) [inline src/components/MessageBox/MessageBox.tsx]: can you please break this line into multiple ones?

**Comment by iVergot** [comment]
@mpisacic what if I want to display message box without status icon (messageLevel enum)?

**Comment by hbagarich** [comment]
@iVergot is this something that we want/anticipate to happen? message boxes have icons?

**Comment by mpisacic** [comment]
@iVergot if you dont provide level as prop you wont get icon

**Comment by iVergot** [comment]
@mpisacic you will, default value for level prop is MessageLevel.Information, you have to pass null to remove the status Icon.
@hbagarich I don't know, that's why i asked 😄

**Comment by mpisacic** [comment]
@iVergot some idea is that message box have that three states, so if you want no icon hack it ;)

**Comment by hbagarich** [comment]
I think we should go with message boxes with icons based on MessageLevel. I think that is expected of the MessageBox component. If the end user really wants no icon, there is hackish solution, but by default the MessageBox should show icons.

**Comment by iVergot** [comment]
since there is close button defined in component properties, should message box close itself on close button click after executing user defined method?

**Comment by mpisacic** [comment]
@iVergot  close button has associated onClose hook (or callback) which defines behavior on close from outside of component, which in general makes this component a controlled one, in some use case I could define that when I press close button some form gets cleared (ie. dispath some action) and then close box with setting its. isOpen prop to false, also we can't just close box on completion because box doesn't know when its associated operation is completed (it is coming from "outside")

**Pull request merged**